.github/workflows/docs.yml

```diff
@@ -1,5 +1,8 @@
 name: PouchDB Docs
 
+env:
+  NODE_VERSION: 14
+
```

Comment:
could this be inline?  is it here for consistency with the other workflow?

Reply:
✅